SignalServiceKit/src/Loki/Utilities/Notification+Loki.swift

```diff
@@ -4,6 +4,7 @@ public extension Notification.Name {
     // State changes
     public static let contactOnlineStatusChanged = Notification.Name("contactOnlineStatusChanged")
     public static let threadFriendRequestStatusChanged = Notification.Name("threadFriendRequestStatusChanged")
+    public static let groupThreadUpdated = Notification.Name("groupThreadLeft")
```

Comment:
Let's make the raw value match

Reply:
Right don't know why xcode didn't pick up that on the refactor ..